go.mod

```diff
@@ -30,10 +30,11 @@ require (
 	golang.org/x/tools v0.0.0-20190909030654-5b82db07426d // indirect
 	google.golang.org/appengine v1.6.1 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
 	k8s.io/api v0.0.0-20190820101039-d651a1528133 // indirect
 	k8s.io/apimachinery v0.0.0-20190823012420-8ca64af22337 // indirect
-	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
+	k8s.io/client-go v0.0.0-20190819141724-e14f31a72a77
```

Comment:
The new client set API in client-go looks very nice. But let’s make sure it’s available in OpenShift 4.2.
OpenShift 4.2 will be based on Kubernetes 1.14. See the corresponding client-go version - https://github.com/kubernetes/client-go/tree/kubernetes-1.14.6 (sha: 7e43eff).

We will need to pin all our kube dependencies to 1.14 version for now. It may break our minishift compatibility though.